docs/architecture/assets.md

```diff
@@ -1,58 +1,81 @@
-In Miden, users can create and trade arbitrary fungible and non-fungible assets.
+# Asset
 
-We differentiate between native and non-native assets in Miden. Native assets follow the Miden asset model. Non-native assets are all other data structures of value that can be exchanged.
+> Fungible and Non-fungible assets in the Miden protocol.
 
-Native assets in Polygon Miden have four goals:
+## What is the purpose of an asset?
 
-* Asset exchange should be parallelizable.
-* Asset ownership should be self-sovereign.
-* Asset usage should be censorship resistant.
-* Fees can be paid using any asset.
+In Miden, assets serve as the primary means of expressing and transferring value between [accounts](accounts.md) through [notes](notes.md). They are designed with four key principles in mind:
 
-All native assets in Miden are stored directly in accounts, like Ether in Ethereum. Miden does not track asset ownership using global hashmaps, e.g., ERC20 contracts. Local asset storage in accounts provides privacy and the ability for client-side proofs. That is because ownership changes are reflected only on an account and not in an ERC20 account (global hashmap). Thus, these changes can happen in parallel. Additionally, asset exchange is censorship resistant at this level because there is no global contract the transfer must pass through. Finally, users can pay fees in any asset.
+1. **Parallelizable exchange:**  
+    By managing ownership and transfers directly at the account level instead of relying on global structures like ERC20 contracts, accounts can exchange assets simultaneously, boosting scalability and efficiency.
```

Comment:
Nit: Instead of "simultaneously" I think I would rather use "parallel" or "concurrently", as those are the more common words to describe this property. This does depend on the target audience though (technical or non-technical). Since I guess it's technical, I would rather use the mentioned words.